### PR TITLE
tokei 6.0.1 (new formula)

### PR DESCRIPTION
**Pull request by myrrlyn**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Add a formula for the `tokei` tool, as it appears to not have one.

I made a mistake on my first commit, and I'm not sure how to squash them here on GitHub; my apologies.

### CODE REVIEW

File: Formula/tokei.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class Tokei < Formula
+  desc "Program that allows you to count code, quickly."
+  homepage "https://github.com/Aaronepower"
+  url "https://github.com/Aaronepower/tokei/releases/download/v6.0.1/tokei-v6.0.1-x86_64-apple-darwin.tar.gz"
```
Comment (ilovezfs) [inline Formula/tokei.rb]: We'll need to build this from source.

**Review by ilovezfs — CHANGES_REQUESTED** [review]

File: Formula/tokei.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "build", "--release"
+    bin.install "target/release/tokei"
+  end
+
+  test do
+    (testpath/"lib.rs").write <<-EOS
```
Comment (dunn) [inline Formula/tokei.rb]: Use `EOS.undent` and indent the Rust code by two spaces.

Comment (myrrlyn) [inline Formula/tokei.rb]: Indent the heredoc by two from the parent level, or use two-space indentation in the code block?

**Comment by dunn** [comment]
Thank you for contributing to Homebrew! Merged as 1141ce59108ddffe82ba4f3b5f9440671212dd0a.

**Pull request closed without merging**